platform/contracts/admin/src/msg/mod.rs

```diff
@@ -70,20 +96,25 @@ pub enum SudoMsg {
     /// it should start as Admin contract migration which would then
     /// continue with the migration of the other contracts.
     MigrateContracts(MigrateContracts),
+    MigrateLeases(BTreeMap<String, MigrateLeases>),
+    MigrateLeasesCont(BTreeMap<String, MigrateLeasesCont>),
```

Comment:
not a good idea due to:
- the introduced knowledge for Leases in the Admin
- broken migration atomicity